src/components/Kv/KvAccordionItem.vue

```diff
@@ -1,30 +1,26 @@
 <template>
 	<div
-		class="kv-accordion"
-		:class="{
-			'kv-accordion--open' : isOpen,
-		}"
+		class="tw-border-b tw-border-tertiary tw-relative"
 	>
 		<button
-			class="kv-accordion__header-button"
+			class="tw-w-full tw-flex tw-justify-between tw-items-center tw-py-[0.75rem] tw-px-0
```

Comment:
Please use a style attribute for custom dimensions such as `tw-py-[0.75rem]` OR convert the size to use the closest measure in our design token/tailwind size matrix.

Reply:
0.75rem is 12px, which is 1.5 is our design token sizes. So `tw-py-1.5` should work.

Reply:
Good, value replaced.